### PR TITLE
Add Webook Reader Examples

### PR DESCRIPTION
**Pull request by danleyb2**
- Flask
- NodeJS with no Framework

### CODE REVIEW

**Review by marcbelmont — COMMENTED** [review]
Can you also update https://github.com/marcbelmont/deep-license-plate-recognition/blob/master/webhooks/README.md ?

File: webhooks/webhook_reader.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        filesParser(request, response, function (err) {
+            if (err instanceof multer.MulterError) {
+                // A Multer error occurred when uploading.
+                console.error(err);
+            } else if (err) {
+                // An unknown error occurred when uploading.
+                console.error(err);
+            }
+
+            // console.log('request files');
```
Comment (marcbelmont) [inline webhooks/webhook_reader.js:70]: Those look like debug comments and should be removed.

File: webhooks/webhook_reader_flask.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        # Files exist for multipart/form-data
+        files = request.files
+        app.logger.debug(f'files: {files}')
+        if files:
+            app.logger.debug('Request contains image')
+
+            for key in files.keys():  # The file doesn't exist under upload
+                app.logger.debug(f'key: {key}')
+                f = files[key]
+                f.save(f'uploads/{f.filename}')
```
Comment (marcbelmont) [inline webhooks/webhook_reader_flask.py:33]: So you have to create the uploads directory before?

File: webhooks/webhook_reader.js (hunk 1 above)
Comment (danleyb2) [reply webhooks/webhook_reader.js]: have removed the comments and added a 500 return on error

File: webhooks/webhook_reader_flask.py (hunk 2 above)
Comment (danleyb2) [reply webhooks/webhook_reader_flask.py]: Yes. have added directory creation if missing. 
This was also missing for the node example

**Comment by danleyb2** [comment]
> Can you also update https://github.com/marcbelmont/deep-license-plate-recognition/blob/master/webhooks/README.md ?

done

**Review by marcbelmont — APPROVED** [review]

**Pull request merged**